### ISSUE TITLE
Manual Endpoint Configuration Required for ZPR service commands in the OCI CLI

### ISSUE THREAD

**Issue by bhagwatvyas** [oracle/oci-cli#849]
**Update: This has been fixed in [v3.49.0](https://github.com/oracle/oci-cli/releases/tag/v3.49.0). If you are not able to upgrade to a non-affected version at this time, please use the workaround mentioned below**

-----------------------------------------------------------------------------------------------------------------------
If you are using or want to use ZPR service commands in OCI CLI then you have to manually update the endpoint.

Description

Users of the OCI CLI who wish to utilize ZPR service commands must manually update the endpoint due to an incorrect configuration in the service endpoint. This issue affects the Zero Trust Packet Routing Service (ZPR) released in version [3.48.2](https://github.com/oracle/oci-cli/releases/tag/v3.48.2).

Workaround

To resolve this issue, users need to manually set the endpoint when executing commands for ZPR service, using the [raw-requests feature](https://docs.oracle.com/en-us/iaas/tools/oci-cli/3.48.2/oci_cli_docs/cmdref/raw-request.html#raw-request). APIs for ZPR can be found [here](https://docs.oracle.com/en-us/iaas/api/#/en/zero-trust-packet-routing/20240301/). Here is an example:
```
oci raw-request --target-uri https://zpr/.{region}.oci.oraclecloud.com/20240301/{api} --http-method {method} --request-body {json-formatted body}
```
Replace {region} with the appropriate region for your OCI deployment.

The correct endpoint for your specific region can be found in the Oracle Cloud Infrastructure API documentation [here](https://docs.oracle.com/en-us/iaas/api/#/en/zero-trust-packet-routing/20240301/).

**Comment by karthik-k-kamath** [comment]
This issue has been resolved in OCI CLI release: [3.49.0](https://github.com/oracle/oci-cli/releases/tag/v3.49.0)

**Comment by karthik-k-kamath** [comment]
Resolved in [3.49.0](https://github.com/oracle/oci-cli/releases/tag/v3.49.0)
